src/ape/managers/project.py

```diff
@@ -29,6 +29,19 @@ def _create_source_dict(contracts_paths: Collection[Path]) -> Dict[str, Source]:
 
 @dataclass
 class ProjectManager:
+    """
+    A singleton in ``ape`` for accessing contract-types and dependencies.
```

Comment:
Not really used as a singleton now that I think about it. There is `project` (the local project) and `Project` (alias to `ProjectManager`)

Reply:
Oh good point!